include/nbl/builtin/hlsl/prefix_sum_blur/box_sampler.hlsl

```diff
@@ -127,36 +138,36 @@ struct BoxSampler
             }
             case ETC_CLAMP_TO_EDGE:
             {
-                result -= (1 - abs(leftIdx)) * prefixSumAccessor.template get<T, uint32_t>(0);
+                result -= (leftIdx + 1) * prefixSumAccessor.template get<T, uint32_t>(0);
                 break;
             }
             case ETC_MIRROR:
             {
                 const T last = prefixSumAccessor.template get<T, uint32_t>(lastIdx);
                 T floored, ceiled;
 
-                if (abs(leftFlIdx + 1) % (2 * linearSize) == 0)
-                    floored = -(abs(leftFlIdx + 1) / linearSize) * last;
+                if (abs(leftFlIdx) % (2 * linearSize) == 0)
+                    floored = -(abs(leftFlIdx) / linearSize) * last;

```

Comment:
you already know `leftFlIdx` is negative, can use `-leftFlIdx` instead of the abs